platform/barefoot/docker-syncd-bfn-rpc/Dockerfile.j2

```diff
@@ -12,20 +12,21 @@ debs/
 RUN apt-get purge -y syncd
 
 ## Pre-install the fundamental packages
-RUN apt-get update \
+RUN apt-get update      \
  && apt-get -y install  \
     net-tools           \
-    python-pip          \
-    python-setuptools   \
+    python3-pip         \
+    python3-setuptools  \
+    python-is-python3   \
     build-essential     \
     libssl-dev          \
     libffi-dev          \
-    python-dev          \
```

Comment:
Are these `*-dev` packages uninstalled at some later point?

Reply:
No

Reply:
Can these packages (at minimum, `python3-dev`, `build-essential`, and `libffi-dev`) be uninstalled at the end, since they aren't needed at runtime?

Reply:
@saiarcot895 , since Docker layer is already created, this will not reduce the final image size, right?

Reply:
Normally yes, but `--squash` is being used to get rid of the intermediate layers, so it will actually reduce disk space.

Reply:
Did not know this. Thank you for the information.